wagtailmenus/tests/test_backend.py

```diff
@@ -1,23 +1,32 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
Can we drop this line as well please? This is only really needed to handle unicode consistently in Python 2, which we don't need to worry about (it still lingers in a few files, but I'll eradicate it in time!) :)